src/styles/App.css

```diff
@@ -0,0 +1,28 @@
+.App {
```

Comment:
I guess you don't need all these styles generated by `create-react-app` anymore.